src/styles/tokens.css

```
@@ -0,0 +1,59 @@
+    --gray-500: #6a7282;
+    --gray-600: #4a5565;
+    --gray-700: #364153;
+
+    /* Black */
+    --black-1000: #000000;
+    --black-950: #0a0a0a;
+    --black-900: #101828;
+    /* white */
+    --white: #ffffff;
+    /* Font */
+    --font-family-main: Pretendard, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)